tests/page-objects/reports/reports.po.js

```diff
@@ -168,12 +166,17 @@ module.exports = {
   },
 
   selectReportNative: async (savedUuids) => {
-    await element(by.css('#reports-list li[data-record-id="' + savedUuids[0] + '"] input[type="checkbox"]')).click();
-    await helper.waitElementToBeVisible(element(by.css('#reports-content .selection-count > span:first-child')), 3000);
+    const checkbox = element(by.css(`#reports-list li[data-record-id="${savedUuids[0]}"] input[type="checkbox"]`));
+    await helper.clickElementNative(checkbox);
+    await helper.waitElementToBeVisibleNative(element(by.css('#reports-content .selection-count > span:first-child')));
     expect(await element.all(by.css(reportBody)).count()).toBe(1);
-    expect(await element(by.css('#reports-content .report-body .item-summary .sender .name')).getText()).toBe('Sharon');
```

Comment:
I removed this expect since we are already waiting for the element's text to match (L 175) and if it doesn't display, then it'll fail. Does it make sense?